src/environment.py

```diff
@@ -0,0 +1,5 @@
+"""Project Global Constants."""
```

Comment:
In my opinion these could belong to the `utils.py` file.

Reply:
Perhaps, at this moment that might make sense, but if either of these two things starts to go, the purpose for the single files might diverge and they would have to be split up again.


In fact, we already have PROJECT_ROOT being used in more than one file `main.py` and `utils.py` which already indicates the need for two files.

We could move the QUERY_PATH into utils since this is the only place its used, but then people would have to go to two different files to see the global configuration....

I am thinking that it is fine the way it is.